psi4/src/psi4/libfunctional/LibXCfunctional.cc

```diff
@@ -91,20 +91,41 @@ LibXCFunctional::LibXCFunctional(std::string xc_name, bool unpolarized) {
         xc_functional_->info->family == XC_FAMILY_HYB_MGGA) {
         /* Range separation? */
         int rangesep = 0;
-        if (xc_functional_->info->flags & XC_FLAGS_HYB_CAM) rangesep++;
-        if (xc_functional_->info->flags & XC_FLAGS_HYB_CAMY) rangesep++;
-        if (xc_functional_->info->flags & XC_FLAGS_HYB_LC) rangesep++;
-        if (xc_functional_->info->flags & XC_FLAGS_HYB_LCY) rangesep++;
-        if (rangesep) {
+        if (xc_functional_->info->flags & XC_FLAGS_HYB_CAMY) {
+            outfile->Printf("Functional '%d' is a HYB_CAMY functional which is not supported in Psi4\n", xc_name.c_str());
+            throw PSIEXCEPTION("HYB_CAMY functionals not supported in Psi4 at present");
+        }
+        if (xc_functional_->info->flags & XC_FLAGS_HYB_LC ) {
+            outfile->Printf("Functional '%d' is a HYB_LC functional which is not supported in Psi4\n", xc_name.c_str());
+            throw PSIEXCEPTION("HYB_LC functionals not supported in Psi4 at present");
+        }
+        if (xc_functional_->info->flags & XC_FLAGS_HYB_LCY) {
+            outfile->Printf("Functional '%d' is a HYB_LCY functional which is not supported in Psi4\n", xc_name.c_str());
+            throw PSIEXCEPTION("HYB_LCY functionals not supported in Psi4 at present");
+        }
+        if (xc_functional_->info->flags & XC_FLAGS_HYB_CAM) {
             lrc_ = true;
-
-            // SR      = LibXC_ALPHA + LibXC_BETA = psi4.set_x_alpha
-            // LR      = LibXC_ALPHA              = psi4.set_x_alpha + psi4.set_x_beta
-            // LR - SR =             - LibXC_BETA =                    psi4.set_x_beta
-
             double alpha, beta;
             xc_hyb_cam_coef(xc_functional_.get(), &omega_, &alpha, &beta);
 
+            /*
+              The values alpha and beta have a different meaning in psi4 and libxc.
+
+              In libxc, alpha is the contribution from full exact
+              exchange, and beta is the contribution from short-range
+              exchange, yielding alpha exact exchange in the long
```

Comment:
shouldn't this be "beta is the contribution difference between short-range and long-range, yielding"

Reply:
Nope, that's how it is in libxc.

Reply:
Ah, I was thinking it was inconsistent with the latter half of the sentence and with the eqns, but now I see it's a syntax issue with "contribution". How about "In libxc, alpha is the amount of exchange in the long range, and beta is the contribution (difference from long range) from short-range exchange."

Reply:
I think there's a problem with signs of beta when you formulate it like that, @loriab. I'd be in favour of a variant of the following:

```
In libxc, alpha is the contribution from exact exchange at all ranges, and beta is the contribution at short range only, yielding alpha exchange only at the long range, and alpha+beta in the short range.

In Psi4, alpha is also the amount of exchange at all ranges, but beta is the difference between the long range and short range, yielding alpha+beta at the long range, and alpha only at the short range.
```

Reply:
I read your version and thought "that's all wrong, too". Then put a filter on the words and it turned all right. So maybe it's just me that's having problems. In any case, the eqn is right, and it's what matters. I'm ok with Susi's original version or yours. So no change needed unless others are having the same trouble.

(Unresolving the conversation so it shows up on GH)